src/cr/cube/dimension.py

```diff
@@ -1233,6 +1239,30 @@ def _order_dict(self):
         return self._dimension_transforms_dict.get("order") or {}
 
 
+class _SmoothingSpec(object):
+    """Value object providing convenient access to details of a smoothing transform."""
+
+    def __init__(self, smoothing_dict):
+        self._smoothing_dict = smoothing_dict
+
+    @lazyproperty
+    def function(self):
+        """String representing the smoothing function specified in the payload.
+
+        Default smoothing function is ``one_sided_moving_avg``
+        """
+        return self._smoothing_dict.get("function", "one_sided_moving_avg")
+
+    @lazyproperty
+    def window(self):
+        """int, the window parameter specified in the smoothing expression.
+
+        The default value (if window is not specified) is 2 for the moving avarage
+        smoothing algorithms.
+        """
+        return self._smoothing_dict.get("window", 2)
```

Comment:
That same approach might be applicable here as well since while `0` is a valid integer, it is outside the range of a valid smoothing window.

```python
return self._smoothing_dict.get("window") or 2
```